### PR TITLE
ForkAdd1: Added my interesting feature to remotefile2.txt

### PR DESCRIPTION
**Pull request by creativeab88**
Hey Abhijith,

This is my very interesting feature for your project/application. Please review and accept.

### CODE REVIEW

File: remotefile2.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 To test fetch option.
+My very interesting feature addition!!!
```
Comment (abhijithvg) [inline remotefile2.txt]: WIll this work. Will have to check. @creativeab88

**Review by abhijithvg — CHANGES_REQUESTED** [review]
Please modify your feature a bit.

File: remotefile2.txt (hunk 1 above)
Comment (creativeab88) [reply remotefile2.txt]: The changes requested are done

**Comment by creativeab88** [comment]
The changes requested are done

**Pull request closed without merging**